src/movies.js

```diff
@@ -1,22 +1,96 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
 
+function getAllDirectors(movies) {
+ const directors = [];
+ movies.forEach(movie => {
+  directors.push(movie.director)
+ });
+ return directors;
```

Comment:
Esto esta perfect! pero podrias hacerlo tambien con un map, asi te ahorras crearte el directors!
```suggestion
function getAllDirectors(movies) {
 return movies.map((movie) => { return movie.director });
 }
```